spec/rack_test_spec.rb

```diff
@@ -145,6 +145,14 @@ module TestSessions
       end
     end
 
+    describe '#send_keys' do
+      it 'raises an UnsupportedMethodError' do
```

Comment:
@twalpole is a new and domain-specific `UnsupportedMethodError` preferable to a `NotImplementedError`?

Reply:
@seanpdoyle I think for the racktest driver instance something like

    raise Capybara::NotSupportedByDriver, "RackTest driver doesn't have the concept of an active element, please call send_keys on a specific element"

probably makes the most sense

Reply:
Thanks! Done.